mobile/status.go

```diff
@@ -66,7 +69,21 @@ type InitializeApplicationResponse struct {
 }
 
 func InitializeApplication(requestJSON string) string {
-	return callWithResponse(initializeApplication, requestJSON)
+	// NOTE: InitializeApplication is logs the call on its own rather than using `callWithResponse`,
+	//       because the API logging is enabled during this exact call.
+	defer callog.Recover(logutils.ZapLogger())
+
+	startTime := time.Now()
+	response := initializeApplication(requestJSON)
+	callog.Log(
+		requestlog.GetRequestLogger(),
+		"InitializeApplication",
+		requestJSON,
+		response,
+		startTime,
+	)
+
+	return response
```

Comment:
```suggestion
	defer callog.Recover(logutils.ZapLogger())
	if err := initializeLogging(requestJSON); err != nil {
	    return makeJSONResponse(err)
	}
	
	return callWithResponse(initializeApplication, requestJSON)
```

Reply:
You want to call `initializeLogging` twice? 🤔

Reply:
No, extract `initializeLogging` from within `initializeApplication`, and call it in `InitializeApplication`.

Reply:
oh, missed that. hmmmmmmm

Reply:
@osmaczko I realised that with your proposal in case of a panic inside `initializeApplication ` we'll get it double-logged.

Reply:
Considering the reduced code complexity and the fact that initializeApplication gets called right at the start, I’d say this is one of those “meh, I can live with it” moments :D

Reply:
@osmaczko you gotta live with this until with switch to grpc 🙂